Octokit.Tests.Integration/Helper.cs

```diff
@@ -8,6 +8,7 @@ public static class Helper
     {
         static readonly Lazy<Credentials> _credentialsThunk = new Lazy<Credentials>(() =>
         {
+            // These environment variables should be set to a test GitHub account using the powershell script configure-integration-tests.ps1
```

Comment:
Should we make this a bit clearer and use xml-doc notation on the `Credentials` property in here?